src/containers/Form/FormLayout.tsx

```diff
@@ -63,6 +63,7 @@ export interface FormLayoutProps {
   showPreviewButton?: boolean;
   onPreivewClick?: Function;
   getQueryFetchPolicy?: any;
+  changePageOnSave?: boolean;
```

Comment:
Maybe change this to
```suggestion
  saveOnPageChange?: boolean;
```

Reply:
Updated